contracts/0.8.25/vaults/StakingVault.sol

```diff
@@ -442,7 +447,7 @@ contract StakingVault is IStakingVault, OwnableUpgradeable {
         if (_ether > valuation_) revert RebalanceAmountExceedsValuation(valuation_, _ether);
 
         ERC7201Storage storage $ = _getStorage();
-        if (owner() == msg.sender || (valuation_ < $.locked && msg.sender == address(VAULT_HUB))) {
+        if (owner() == msg.sender || (valuation_ < $.locked && msg.sender == address(VAULT_HUB) && $.vaultHubAuthorized)) {
```

Comment:
I'd simplify it by splitting into multiple cases tbh

Reply:
added early revert as for TW